### DISCUSSION TITLE
From calorimeter data to FDS input for sugar

### DISCUSSION THREAD

**Discussion by Zaninho23** [firemodels/fds#12906] [Q&A]
Good morning, everyone. I'm new to FDS and I have a question about determining the HRR curve to be included within the calculation model from the results obtained at the calorimetric cone. How can I use the results of the test to determine the HRR curve to be entered into FDS? The sample tested had dimensions of 100x100x25mm, while the sugar, in my case, was deposited in bags of about 1x1x1m. How can I determine the HRRmax and the trend of the curve for the bag, given that the dimensions are completely different from that of the sample? Should I build an ad hoc FDS model by simulating the calorimetric cone test and then set the material characteristics in the general model? Or should I do it another way/simplier way?
Thank you very much.

**Comment by mcgratta** [comment]
If you have the latest version of FDS, search the User's Guide for "Scaling Pyrolysis".

**Reply by Zaninho23** [reply]
thank you very much, I will study it.

**Comment by Zaninho23** [comment]
good morning, I studied section 9.1.4 of the manual and built on the basis of the file spyro_cone_demo.fds the file you find attached in which I entered the parameters related to sugar that I found in the literature (calorimetric cone test with flux equal to 35kW/m2 test size 100x100x25mm and results you see in the graph below for sucrose) 
![image](https://github.com/firemodels/fds/assets/38040558/a494a7e7-e085-422f-9a14-33beb9b88db6)
I then tried on the basis of the data from the calorimetric cone test to vary the thickness of the material and it seems to work. The results obtained are as follows for thicknesses of 0.025m;0.05m,0.1m: 
![image](https://github.com/firemodels/fds/assets/38040558/ebb42731-1ffc-4d85-8258-013ff31b2144)
Are these results in your experience correct (all tests reach 600kW/m2 regardless of thickness) ?
What I have not figured out, however, unfortunately, is how to implement this model in a more structured simulation, such as a scenario where the 1x1x1m bag of sugar is burning. Do you have any suggestions for this ?
In the meantime, I do some testing in FDS
[spyro_cone_demo_Sugar_bags.txt](https://github.com/user-attachments/files/15511265/spyro_cone_demo_Sugar_bags.txt)

Many many thanks !!!

**Reply by drjfloyd** [reply]
That behavior is how the scaling model is constructed (details in the Technical Reference Guide appendices). Going from an 0.025 cm test to 1 m is extrapolating by a factor of 40. Extrapolation errors typically scale up with the amount of extrapolation. You also have a very different geometry. A cone test would have the sugar confined in a steel tray. You have a bag which might fail at some point allowing the sugar to spill and increase in area.  Sugar melts and the impact of that in a 1 m bag may differ to some extent from a shallow tray. You may need to make some engineering judgement on how to best handle this.

**Reply by johodges** [reply]
Looking at the paper where the data is from, the authors included this statement based on the results. I am hesitant to use these data for thickness scaling since it is not fully established. It would probably be better to try and come up with a kinetics representation of the burning.

![image](https://github.com/firemodels/fds/assets/17127980/fd8c1e17-5b2a-4e77-8b89-b2879211aa2d)

https://www.aidic.it/cet/19/77/036.pdf

**Reply by Zaninho23** [reply]
Thank you very much for your suggestions. 
In particular, my goal is to determine the HRRmax released from the combustion of my sugar bag. (with HRRmax afterwards I can do the complete model of my stock considering a solid thick material burning at a specific rate - manual chapter 9.1.3)
That's why I researched the calorimetric cone data and tried pyrolysis scaling. Thanks to you, however, I realized that there is too much difference between the size of my bag and the test sample. The error, as you told me, is greater the larger the scaling are. Sugar melting is also a problem.
At this point according to what you have written I could try to determine the HRRmax:
1. Based on engineering evaluations 
2. Using an FDS model with a kinetics representation of the burning

1- In the first case do you think I could determine the peak from the peakHRR value of the test by multiplying it by the combustible surface of my bag ? 
- Peak HRR (cone test) =600kW/m2 
-burnable surface of the bag= 5 sides of the bag 1x1m2 =5m2
- HRRmax (for one bag)= 600kW/m2*5m2=3000kW (considering one bag with 1m2 of floor occupation)
(Neglecting that the sack at a certain temperature will break and let out the sugar which will then take a conical shape and not a cubical shape (surface of cubical shape > surface of conical shape). 
Neglecting also the sugar melting which I would not know how to consider)

2- In the second case on the other hand if I wanted to describe the combustion of my sugar bag usinag a FDS model I would have to refer to what is described within the manual in chapter 9.2 (Complex Pyrolysis Model), but it seems very complicated to me.

Thank you very much and sorry for so many questions, in FDS and Fire safety Engineering I am a beginner.
